content/en/docs/concepts/cluster-administration/logging.md

```diff
@@ -67,70 +78,74 @@ kubectl logs counter -c count
 
 When you run [`kubectl logs`](/docs/reference/generated/kubectl/kubectl-commands#logs) as in
 the basic logging example, the kubelet on the node handles the request and
 reads directly from the log file. The kubelet returns the content of the log file.
 
+
 {{< note >}}
-If an external system has performed the rotation or a CRI container runtime is used,
-only the contents of the latest log file will be available through
-`kubectl logs`. For example, if there's a 10MB file, `logrotate` performs
-the rotation and there are two files: one file that is 10MB in size and a second file that is empty.
-`kubectl logs` returns the latest log file which in this example is an empty response.
+Only the contents of the latest log file are available through
+`kubectl logs`.
+
+For example, if a Pod writes 40 MiB of logs and the kubelet rotates logs
+after 10 MiB, running `kubectl logs` returns at most 10MiB of data.
 {{< /note >}}
 
-### System component logs
+## System component logs
 
 There are two types of system components: those that run in a container and those
 that do not run in a container. For example:
 
 * The Kubernetes scheduler and kube-proxy run in a container.
 * The kubelet and container runtime do not run in containers.
 
-On machines with systemd, the kubelet and container runtime write to journald. If
-systemd is not present, the kubelet and container runtime write to `.log` files
-in the `/var/log` directory. System components inside containers always write
-to the `/var/log` directory, bypassing the default logging mechanism.
-They use the [`klog`](https://github.com/kubernetes/klog)
-logging library. You can find the conventions for logging severity for those
-components in the [development docs on logging](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md).
+On machines with systemd, the kubelet and container runtime write to journald.
+If systemd is not present, the kubelet and container runtime write to `.log` files in the
+`/var/log` directory.
+System components inside containers always write to the `/var/log` directory,
+bypassing the default logging mechanism.
 
-Similar to the container logs, system component logs in the `/var/log`
-directory should be rotated. In Kubernetes clusters brought up by
-the `kube-up.sh` script, those logs are configured to be rotated by
-the `logrotate` tool daily or once the size exceeds 100MB.
+{{< note >}}
+If you deploy Kubernetes cluster components (such as the scheduler) to log to
+a volume shared from the parent node, you need to consider and ensure that those
+logs are rotated. Kubernetes does not manage that log rotation. Your operating
+system may automatically implement some log rotation - for example, if you share
+the directory `/var/log` into a static Pod for a component, node-level log rotation
+treats a file in that directory the same as a file written by any component outside
+Kubernetes.
```

Comment:
Use [simple and direct language](https://kubernetes.io/docs/contribute/style/style-guide/#use-simple-and-direct-language)
```suggestion
Kubernetes does not manage log rotation. If you deploy Kubernetes cluster components (such as the scheduler) to log to
a volume shared from the parent node, make sure that those
logs are rotated. 

Your operating system may automatically implement some log rotation. 
For example, if you share the directory `/var/log` into a static Pod for a component, 
node-level log rotation treats a file in that directory the same as a file written
by any component outside Kubernetes.
```

Reply:
You know what, CRIContainerLogRotation is GA so I think this needs a bigger change.

Reply:
I think that's sorted, now.